boa/src/builtins/array/tests.rs

```diff
@@ -675,3 +675,69 @@ fn for_each_push_value() {
     assert_eq!(forward(&mut engine, "a[6]"), "6");
     assert_eq!(forward(&mut engine, "a[7]"), "8");
 }
+
```

Comment:
@Nickforall you don't have a #[test] attribute above the filter() function here like the others

Reply:
Whoops, must have happened while fixing merge conflicts